src/argilla_server/schemas/v1/questions.py

```diff
@@ -61,63 +58,58 @@
 
 SPAN_OPTIONS_MIN_ITEMS = 1
 SPAN_OPTIONS_MAX_ITEMS = 500
+SPAN_MIN_VISIBLE_OPTIONS = 3
```

Comment:
We don't want a maximum value for this?

Reply:
the max limit is validated by the custom pydantic validator. This is the same logic as we have for single and multi-label questions.

Reply:
Ok, I missed that you added the validator for that.